src/python/system/environment.py

```diff
@@ -971,7 +971,7 @@ def set_value(environment_variable, value):
   """Set an environment variable."""
   value_str = str(value)
   environment_variable_str = str(environment_variable)
-  value_str = value_str.replace('%ROOT_DIR%', os.environ['ROOT_DIR'])
+  value_str = value_str.replace('%ROOT_DIR%', os.getenv('ROOT_DIR', ''))
```

Comment:
SHould this be empty string or "."

Reply:
For libClusterFuzz, we don't expect ROOT_DIR to be set at all, so have the default of ''.